docs/source/json-api/index.rst

```diff
@@ -32,140 +32,123 @@ We welcome feedback about the JSON API on `our issue tracker
-            Optional websocket configuration string. Contains comma-separated key-value pairs. Where:
-            maxDuration -- Maximum websocket session duration in minutes
-            heartBeatPer -- Server-side heartBeat interval in seconds
-            Example: "maxDuration=120,heartBeatPer=5"
-
-With Authentication
-===================
+With Query Store
+------------------
 
-Apart from interacting with the Ledger API on behalf of the user, the HTTP JSON API server must also interact with the Ledger API to maintain some relevant internal state.
+Often when running the JSON API you'll want to cache queries, especially large queries, to decrease the workload of your underlying ledger.
+The JSON API provides a way to do this by allowing you to specify a database to cache queries (currently PostgreSQL only).
 
-For this reason, you must provide an access token when you start the HTTP JSON API if you're running it against a Ledger API server that requires authentication.
+In this case you can use the ``--query-store-jdbc-config`` flag, an example of which is below. 
 
-Note that this token is used exclusively for maintaining the internal list of known packages and templates, and that it will not be use to authenticate client calls to the HTTP JSON API: the user is expected to provide a valid authentication token with each call.
+.. note:: When you use the Query Store you'll want your first run to specify ``createSchema=true`` so that all the necessary tables are created. After the first run make sure ``createSchema=false`` so that it doesn't attempt to create the tables again.
 
-The HTTP JSON API server requires no access to party-specific data, only access to the ledger identity and package services. A token issued for the HTTP JSON API server should contain enough claims to contact these two services but no more than that. Please refer to your ledger operator's documentation to find out how.
+.. code-block:: shell
 
-Once you have retrieved your access token, you can provide it to the HTTP JSON API by storing it in a file. Give the path to it with the ``--access-token-file`` command line option.
+    daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575 \
+    --query-store-jdbc-config "driver=org.postgresql.Driver,url=jdbc:postgresql://localhost:5432/test?&ssl=true,user=postgres,password=password,createSchema=false"
 
-If the token cannot be read from the provided path or the Ledger API reports an authentication error (for example due to token expiration), the HTTP JSON API will report the error via logging. The token file can be updated with a valid token, and it will be picked up at the next attempt to send a request.
+.. note:: The JSON API provides many other useful configuration flags, run ``daml json-api --help`` to see all of them.
 
-Example session
-***************
+Authentication and Authorization
```

Comment:
I don’t think this section is correct.
1. Authentication isn’t handled by the ledger API. It is handled by your IAM, e.g., in our auth blogpost auth0 takes care of that.
2. The tokens are called access tokens not authentication tokens and authorization tokens or at least I have never heard that being used and it doesn’t match what we use in https://docs.daml.com/app-dev/authorization.html.
3. Related to that, I don’t understand why getting packages should be authentication and user requests are authorization. I think the crucial point here is that one request is initiated by the JSON API itself and it needs its own token for that (this needs documentation on which claims are required here) whereas the user-initiated requests just proxy the token from users.

Reply:
1. Fixed.
2. They are access tokens but they're used for Authentication and Authorization. This was an attempt to disambiguate which access token the docs are talking about as it's a regular source of confusion. In hindsight leveraging these terms was the wrong approach. Let me know if the new commit makes sense and if there's any other changes to this section.